Example

Unable to run the example. Build error

Example:
```golang
package main

import (
    "github.com/gogpu/gogpu"
    "github.com/gogpu/gogpu/gmath"
)

func main() {
    app := gogpu.NewApp(gogpu.DefaultConfig().
        WithTitle("Hello GoGPU").
        WithSize(800, 600))

    app.OnDraw(func(dc *gogpu.Context) {
        dc.DrawTriangleColor(gmath.DarkGray)
    })

    app.Run()
}
```

Error:
```bash
Build Error: go build -o /Users/kiko/Desktop/Naruto/Narurogue/cmd/gg/__debug_bin2583774677 -gcflags all=-N -l .
# github.com/go-webgpu/goffi/ffi
/Users/xxx/go/pkg/mod/github.com/go-webgpu/goffi@v0.3.9/ffi/cgo_unsupported.go:28:9: undefined: GOFFI_REQUIRES_CGO_ENABLED_0 (exit status 1)
```

--- a/README.md
+++ b/README.md
@@ -50,11 +50,14 @@ go get github.com/gogpu/gogpu
 
 **Requirements:**
 - Go 1.25+
+- `CGO_ENABLED=0` (Pure Go FFI requires CGO disabled)
 
 **Zero dependencies — just works:**
 ```bash
-go run .
+CGO_ENABLED=0 go run .
 ```
+
+> **Note:** On macOS and some Linux distros, CGO is enabled by default. Always set `CGO_ENABLED=0` when building GoGPU projects.
 
 ---
 

